src/portable/portable.c

```diff
@@ -951,11 +951,98 @@ static int make_marker_text(const char *image_path, OrderedHashmap *extension_im
         return 0;
 }
 
+/* 'output=' is always appended to, even if there are ENOMEM errors */
+static int append_release_info_to_unit(char **output, const PortableMetadata *release, bool sysext_prefix) {
```

Comment:
hmm, maybe use ImageClass for the 3rd arg, instead of "bool". I figure sooner or later we want this for the syscfg stuff anyway, too. And using booleans for choosing between two sets of names is a bit icky anyway